openshift/release/update-to-head.sh

```diff
@@ -38,7 +38,12 @@ git commit -m ":robot: Triggering CI on branch 'release-next' after synching to
 git push -f openshift release-next-ci
 
 if hash hub 2>/dev/null; then
-   hub pull-request --no-edit -l "kind/sync-fork-to-upstream" -b openshift-knative/${REPO_NAME}:release-next -h openshift-knative/${REPO_NAME}:release-next-ci
+   # Test if there is already a sync PR in 
+   COUNT=$(hub api -H "Accept: application/vnd.github.v3+json" repos/openshift/${REPO_NAME}/pulls --flat \
```

Comment:
This command should end with `|| true` and the condition below should have a single `=`, see the same fixes for knative-serving:
https://github.com/openshift/knative-serving/pull/798
https://github.com/openshift/knative-serving/pull/799